.github/workflows/build.yml

```diff
@@ -113,6 +113,55 @@ jobs:
         git -C taps/ push
 
 
+  build_unix_arm:
+      needs: prepare
+      runs-on: ubuntu-18.04  # Standalone executable should be built on minimum supported OS
+      strategy:
+        matrix:
+          architecture:
+          - armv7
+          - aarch64
+
+      steps:
+      - uses: actions/checkout@v3
+        with:
+          path: ./repo
+
+      - name: Virtualized Install, Prepare & Build
+        uses: uraimo/run-on-arch-action@v2
+        with:
+          arch: ${{ matrix.architecture }}
+          distro: ubuntu18.04
+
+          setup: |
+            mkdir -p "${PWD}/artifacts"
+
+          # Setup shared volume folders between the host container and the child container
+          dockerRunArgs: |
+            --volume "${PWD}/repo:/repo"
+            --volume "${PWD}/artifacts:/artifacts"
+            
+          # Additional shell commands to be executed in the docker container as part of docker build. Will be included in the cached docker image if githubToken is set 
+          install: |
+            apt update
+            apt -y install zlib1g-dev python3.8 python3.8-dev python3.8-distutils python3-pip
```

Comment:
1. Where's `zlib1g-dev` used?
2. Why 3.8? If there is no specific reason, use 3.10

Reply:
1. python3.8-dev depends on zlib1g-dev
2. Attempting to install 3.10 from the deadsnakes repo raised some errors, so I opted for using 3.8 instead

Reply:
Add a comment with reason for 3.8

Reply:
Done